include/SFML/Window/Vulkan.h

```diff
@@ -28,6 +28,21 @@
 #include <SFML/Window/Export.h>
 
 
+typedef struct VkInstance_T* VkInstance;
```

Comment:
Where is `VkInstance_T` defined?

Reply:
This is the line where's defined. I copied this out of SFML itself. I think it's defining a struct named `VkInstance_T` then immediately defining a pointer to that struct named `VkInstance`.

Reply:
Wait sorry I see what you mean now. This is just a forward declaration. Good question. I'm not sure where it's defined as a matter of fact.

Reply:
https://android.googlesource.com/platform/frameworks/native/+/b1352bc/vulkan/libvulkan/loader.cpp#35

I found it defined inside some Vulkan stuff. Not sure if this is an authoritative source but it seems to imply that this structure is something defined by Vulkan.